www/front_src/src/Resources/Actions/Resource/Acknowledge/Dialog.tsx

```diff
@@ -82,8 +82,7 @@ const DialogAcknowledge = ({
             label={labelComment}
             fullWidth
             rows={3}
-            error={errors?.comment !== undefined}
-            helperText={errors?.comment}
```

Comment:
helper text is directly manage by TextField component ?

Reply:
Yes, this was done in the latest centreon-ui PR, error is now either a string or undefined which simplifies its use.